src/utils.ts

```diff
@@ -78,69 +68,53 @@ export function debounce(fn, ms) {
 /**
  * Prevents errors from being thrown while accessing nested modifier objects
  * in `popperOptions`
- * @param {Object} obj
- * @param {String} key
- * @return {Object}
  */
-export function getModifier(obj, key) {
+export function getModifier(obj: any, key: string) {
   return obj && obj.modifiers && obj.modifiers[key]
 }
 
 /**
  * Determines if an array or string includes a value
- * @param {any[]|String} a
- * @param {any} b
- * @return {Boolean}
  */
-export function includes(a, b) {
+export function includes(a: any[] | string, b: any): boolean {
   return a.indexOf(b) > -1
 }
 
 /**
  * Determines if the value is singular-like
- * @param {any} value
- * @return {Boolean}
  */
-export function isSingular(value) {
+export function isSingular(value: any): boolean {
   return (
     !!(value && hasOwnProperty(value, 'isVirtual')) || value instanceof Element
   )
 }
 
 /**
  * Firefox extensions don't allow setting .innerHTML directly, this will trick it
- * @return {String}
  */
 export function innerHTML() {
```

Comment:
```suggestion
export function innerHTML(): 'innerHTML' {
```

This will fix those ugly `;(element as any)[innerHTML()]` things.